docs/config-definition.md

```diff
@@ -53,11 +49,6 @@ The configurable parameters are as follows:
 | `backupStorageProvider/name` | String<br><br>(Ark natively supports `aws`, `gcp`, and `azure`. Other providers may be available via external plugins.) | Required Field | The name of the cloud provider that will be used to actually store the backups. |
 | `backupStorageProvider/bucket` | String | Required Field | The storage bucket where backups are to be uploaded. |
 | `backupStorageProvider/config` | map[string]string<br><br>(See the corresponding [AWS][0], [GCP][1], and [Azure][2]-specific configs or your provider's documentation.) | None (Optional) | Configuration keys/values to be passed to the cloud provider for backup storage. |
-| `backupSyncPeriod` | metav1.Duration | 60m0s | How frequently Ark queries the object storage to make sure that the appropriate Backup resources have been created for existing backup files. |
```

Comment:
I'm thinking that we should add a section to this page that states that `ark server` has a number of config flags.  We can include a brief snippet of YAML that shows how to specify them, and then put a link to the `ark server` CLI reference docs for the details of the flags

Reply:
Ah I read the comment below first. I think this is a great idea. In this case I'm good with removing the flag in the deployment files.